owasp-top10-2017-apps/a1/sstype/src/public/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>SSType - Hello {{ name }}</title>
+    <link rel="icon" href="./public/images/favicon.png">
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <style>
+        .center {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+        }
+        body {
+            background-image: url('https://www.thomsonfinancialpartners.ca/wp-content/uploads/2017/01/gradient-bg.jpg')
```

Comment:
I am not sure if we can use this image due to its LICENSE. Also, if the link is changed or broken somehow, the application will not be able to render it. My suggestion is replacing it with a similar background color. What do you think? 🙃

Reply:
I changed the background image to a background gradient color with CSS